src/tests/multiplexer-integration-test.ts

```diff
@@ -86,11 +116,15 @@ describe('Multiplexer', () => {
         .expectRenderSlot('List', 'root', {contentTypes: ['templateName', 'model']})
         .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']})
         .expectRenderSlot('ShowOne', 'item', {contentTypes: ['templateName', 'model']})
-        .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']});
+        .expectRenderSlot('PostMuxer', 'item', {contentTypes: ['templateName', 'model']})
+        // Current implementation of multiplexer is to update all the existing hosted particle inputs.
```

Comment:
could you, please, add a TODO here (you can assign it to me) to fix the multiplexer particle implementation to not re-render everything? thanks.